lib/GADS/Datum/Person.pm

```diff
@@ -265,11 +267,9 @@ sub html
         $t = encode_entities $t;
         push @details, $site->register_freetext2_name.": $t";
     }
-    my $details = join '<br>', map {encode_entities $_} @details;
+    my $details = join '<br>', @details;
```

Comment:
@abeverley I'd like it to be - but would prefer to factor out the `html`, `withlinks` and `form_html` routines in a separate PR, so this one doesn't become too large.

The coming PR would be using the presentation layer in other data templates as well, not just the data table, which currently isn't possible since they still rely on the old structure coming from the controller.